### DISCUSSION TITLE
2.6.3 Build 221022-1

### DISCUSSION THREAD

**Discussion by TGSAN** [TGSAN/CMWTAT_Digital_Edition#49] [Announcements]
## 中文版:

1. 支持 ARM64 架构的 Windows 10/11（实验性）
2. 增加了一键将核心版本 Windows 升级到完整版 Windows 的按钮（只会在核心版本 Windows 上可见）
3. “收集低级操作系统的激活状态” 模块升级到 AnyCPU 版本（现已支持ARM64处理器，但是可能会误报为病毒）
4. 法语本地化语言文件更新（半成品）
5. 依赖库升级到最新版（包含Newtonsoft.Json的安全漏洞修复）
6. UI 界面微调
7. 优化代码

## English Version:

1. Support Windows 10/11 on ARM64 (Experimental)
2. Added a one-click button to upgrade the core version of Windows to the full version of Windows (will only be visible on the core version of Windows)
3. "Gather Downlevel OS Activation State" module upgraded to AnyCPU version (now supports ARM64 processors, but may be falsely reported as a virus)
4. French language localization file updated (WIP).
5. Dependency library upgrade to latest version (includes security vulnerability fix for Newtonsoft.Json)
6. UI fine-tuning
7. Code optimization

## 中文版:

1. 修复 Windows 11 22H2 及更新的 Windows on ARM64 系统上无法激活的问题

## English Version:

1. Fixes an issue that prevents activation on Windows 11 22H2 and newer Windows on ARM64

## 中文版:

1. 修复 ARM64 部分 SKU 的问题
2. 固定版本“客户端许可平台迁移工具”升级至 SDK 10.0.20348.0

## English Version:

1. Fix issue on 22H2 ARM64 SKUs.
2. Update fixed "Client License Platform migration tool" version to SDK 10.0.20348.0。

## 中文版:

1. 修复 Windows 10 激活报错问题

## English Version:

1. Fix Windows 10 activation error.

#24 #30 #37 #43 #45 

<hr /><em>This discussion was created from the release <a href='https://github.com/TGSAN/CMWTAT_Digital_Edition/releases/tag/2.6.3.0'>2.6.3 Build 221022-1</a>.</em>

**Comment by ddcatgg** [comment]
这个版本运行时会被win11系统自带的安全软件拦截，版本2.5.0 不会。

**Reply by TGSAN** [reply]
#63 2.6.4 版本发布了，刚发布应该没有被拦截
